### ISSUE TITLE
Pip install fails on recent python versions

### ISSUE THREAD

**Issue by CJones-Optics** [vikhyat/moondream#168]
# Problem
When trying to install moondream in python3.13 on Fedora via pip fails. 

```
  ERROR: Failed building wheel for onnx
Failed to build onnx
ERROR: ERROR: Failed to build installable wheels for some pyproject.toml based projects (onnx)
```

This problem doesn't appear when reverting to earlier version of pythonn

# System
- Python Version 3.13.0
- OS: Fedora Linux 41 (Workstation Edition)

# Workaround
- Create a virtual environment for python 3.10.15
- Run pip install as normal

**Comment by ransheng11** [comment]
What's your numpy version? I am using python=3.10. I can install moondream, but the numpy version conflicts with other packages.

**Comment by CJones-Optics** [comment]
It was a fresh virtual environment, with no other packages installed.

**Comment by Carl-Lea** [comment]
I encountered the same issue with Python 3.13.1. It appears that the unsuccessful installation of onnxruntime was the primary cause. After reviewing information on onnxruntime, I found that it may encounter installation problems with newer Python versions. To resolve this, I downgraded my Python version to 3.12.7, which successfully solved the problem.

**Comment by DJCool1** [comment]
TLDR for it to just work in Python 3.13: `pip install onnx-weekly`
